Client/src/Unified/Services/Submitter/BatchUntilInactiveBlock.cs

```diff
@@ -0,0 +1,249 @@
+  }
+
+  /// <summary>
+  ///   Simple Getter to return size of batch in the pipeline
+  /// </summary>
+  public int BatchSize
+    => source_.BatchSize;
+
+  /// <summary>
+  ///   Return the TimeSpan timer set in the constructor
+  /// </summary>
+  public TimeSpan Timeout { get; }
+
+  /// <summary>
+  ///   Task to check the completion
+  /// </summary>
+  public Task Completion
+    => source_.Completion;
+
+  /// <summary>
+  ///   Signal that the pipeline should be closed
+  /// </summary>
+  public void Complete()
+    => source_.Complete();
+
+  /// <summary>
+  ///   Return the exception produce in a Transform block or ActionBlock
+  /// </summary>
+  /// <param name="exception"></param>
+  public void Fault(Exception exception)
+    => ((IDataflowBlock)source_).Fault(exception);
+
+  /// <summary>
+  ///   Link block TransformBLock or ActionBlock
+  /// </summary>
+  /// <param name="target">the Block to link to</param>
+  /// <param name="linkOptions">
+  ///   Specific option to configure max message and propagation completion
+  ///   https://learn.microsoft.com/fr-fr/dotnet/api/system.threading.tasks.dataflow.dataflowlinkoptions?view=net-6.0
+  /// </param>
+  /// <returns></returns>
+  public IDisposable LinkTo(ITargetBlock<T[]>   target,
+                            DataflowLinkOptions linkOptions)
+    => timeoutTransformBlock_.LinkTo(target,
+                                     linkOptions);
+
+  /// <summary>
+  ///   To signal to the propagator object that a message is coming.
+  ///   Check if the pipeline is opened and the message is accepted
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="messageValue"></param>
+  /// <param name="source"></param>
+  /// <param name="consumeToAccept"></param>
+  /// <returns></returns>
+  public DataflowMessageStatus OfferMessage(DataflowMessageHeader messageHeader,
+                                            T                     messageValue,
+                                            ISourceBlock<T>       source,
+                                            bool                  consumeToAccept)
+  {
+    var offerResult = ((ITargetBlock<T>)source_).OfferMessage(messageHeader,
+                                                              messageValue,
+                                                              source,
+                                                              consumeToAccept);
+
+    if (offerResult == DataflowMessageStatus.Accepted)
+    {
+      timer_.Change(Timeout,
+                    System.Threading.Timeout.InfiniteTimeSpan);
+    }
+
+    return offerResult;
+  }
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock TInput  to accept and consume a
+  ///   DataFlowMessageHeader previously offered by this ISourceBlockTOutput .
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  /// <param name="messageConsumed"></param>
+  /// <returns></returns>
```

Comment:
You need a description for all the params
Maybe you can use directly the following:
```c#
/// <inherit_doc/>
```